test/ModuleInterface/if-configs.swift

```diff
@@ -89,3 +89,39 @@ public func hasClosureDefaultArgWithSinglePoundIf(_ x: () -> Void = {
 // CHECK: func hasSimpleDefaultArgs(_ x: Int = 0, b: Int = 1)
 public func hasSimpleDefaultArgs(_ x: Int = 0, b: Int = 1) {
 }
+
+// CHECK: func hazArrayDefaultArg(_ x: [Int] = [
+// CHECK-NEXT:   1, 2,
+// CHECK-NOT: #if NOT_PROVIDED
+// CHECK-NOT:   3,
+// CHECK-NOT: #endif
+// CHECK-NEXT: 4,
+// CHECK-NOT: #if !NOT_PROVIDED
+// CHECK-NOT:   #if false
+// CHECK-NOT:     50,
+// CHECK-NOT:   #else
+// CHECK:         5,
```

Comment:
Did a blank line end up in here? That's not the end of the world, but it would be nice to know.

Reply:
No reason.. I’ve changed it to a CHECK-NEXT:. 

Thanks for the Review. We must be nearly there. The nits are ready to push once you’ve finished testing.